docs/integrations/sources/mssql.md

```diff
@@ -26,14 +26,16 @@ MSSQL data types are mapped to the following data types when synchronizing data:
 
 If you do not see a type in this list, assume that it is coerced into a string. We are happy to take feedback on preferred mappings.
 
+Please see [this issue](https://github.com/airbytehq/airbyte/issues/4270) for description of unexpected behaviour for certain datatypes.
```

Comment:
Are decimal/numeric/money/smallmoney all working properly now with the property setting? If so I can edit this issue to remove them as problematic types

Reply:
@Phlair the test cases in `MssqlSourceComprehensiveTest` are now passing with the right values for these data types

Reply:
Just realised I made this comment on the wrong issue link. I'll edit [the correct issue](https://github.com/airbytehq/airbyte/issues/4542) accordingly, there's still some other type issues worth explaining.